executor/cook/executor.py

```diff
@@ -305,12 +302,11 @@ def manage_task(driver, task, stop_signal, completed_signal, config):
         task_completed_signal = Event() # event to track task execution completion
         sequence_counter = cp.ProgressSequenceCounter()
 
-        def send_progress_message(message):
-            return send_message(driver, message, config.max_message_length)
-
+        send_progress_message = functools.partial(send_message, driver)
```

Comment:
I like `def` or `lambda` better

Reply:
Keeping this as `partial` as it aligns with our Clojure style.